tsconfig.json

```diff
@@ -1,29 +1,30 @@
-{
-    "version": "2.1.1",
-    "compilerOptions": {
-        "lib": ["es5", "es6"],
-        "outDir": "build/compiled",
-        "target": "es5",
-        "module": "commonjs",
-        "moduleResolution": "node",
-        "importHelpers": true,
-        "emitDecoratorMetadata": true,
-        "experimentalDecorators": true,
-        "sourceMap": true,
-        "noImplicitAny": true,
-        "declaration": true,
-        "noFallthroughCasesInSwitch": true,
-        "noImplicitReturns": true,
-        "stripInternal": true,
-        "pretty": true,
-        "strictNullChecks": true,
-        "noUnusedLocals": true,
-        "downlevelIteration": true
-    },
-    "exclude": [
-        "tmp",
-        "temp",
-        "build",
-        "node_modules"
-    ]
-}
+{
+    "version": "2.1.1",
+    "compilerOptions": {
+        "lib": ["es5", "es6"],
+        "outDir": "build/compiled",
+        "target": "es5",
+        "module": "commonjs",
+        "moduleResolution": "node",
+        "skipLibCheck": true,
```

Comment:
can you please explain this option? Does it really bring good performance in? I remember I had some issues with this option

Reply:
@pleerock Hi! This is only to avoid trying to compile the code with the definitions in `node_modules` as the project is using TS 2.9 and some type definitions (I think these are those from `yargs`) are using things from TS 3+ (`unknown` etc). I can remove it if you're not bothered by the same issue on your machine / in the CI (I assume so, as it is passing)